Subject: Relax bitfield dependency
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

bitfield's changes from 0.14 to 0.17 don't seem to have a significant
impact on the way that tss-esapi uses bitfield, so we can use the
newer version safely.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,7 +33,7 @@
 repository = "https://github.com/parallaxsecond/rust-tss-esapi"
 
 [dependencies.bitfield]
-version = "0.14.0"
+version = "0.17"
 
 [dependencies.enumflags2]
 version = "0.7.1"
